src/file_organizer/cli/main.py

```
@@ -157,19 +265,99 @@ def launch_tui() -> None:
+
+    if build:
+        console.print("[bold]Building documentation to HTML...[/bold]")
+        proc = subprocess.run([mkdocs, "build"], cwd=project_root)
+        if proc.returncode != 0:
+            console.print("[red]Error: Failed to build documentation.[/red]")
+            raise typer.Exit(code=1)
+        console.print(f"[green]Documentation built successfully in {project_root / 'site'}[/green]")
+    else:
+        console.print(f"[bold]Starting documentation server[/bold] at http://{host}:{port}/")
+        try:
+            subprocess.run(
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)